frontend/components/ProjectStatusIndicator.tsx

```diff
@@ -0,0 +1,29 @@
+import { ProjectStatus } from "@/utils/types"
+import type { JSX } from "@emotion/react/jsx-runtime"
+import ErrorOutlineIcon from "@mui/icons-material/ErrorOutline"
+
+export function ProjectStatusIndicator({ s }: { s: ProjectStatus }) {
+  const statusConfig: Record<ProjectStatus, JSX.Element | null> = {
+    [ProjectStatus.RUNNING]: (
+      <div className="flex space-x-1 items-center justify-center">
+        {[0, 150, 300].map((delay) => (
+          <div
+            key={delay}
+            className="w-1.5 h-1.5 rounded-full bg-green-700 animate-bounce"
+            style={{ animationDelay: `${delay}ms` }}
+          />
+        ))}
```

Comment:
I think you can solve that without react at this point. Just by using tailwindcss.

Reply:
No just meant the iteration over 0, 150, 300, but all good. Dont worry about it.